### DISCUSSION TITLE
comment为null时,markdown转换报500

### DISCUSSION THREAD

**Discussion by Andy-njupt** [walinejs/waline#1848] [Ideas]
我在用postman调试提交评论的接口时，payload没有给任何东西，response虽然500报错，但是数据库中存了这样一条comment为null的数据。在ui后台，调用get评论的接口时markdown转换过程中应该是因无法解析null，而报了Input data should be a String，500的error；这里是否可以做个兼容。
另外，前端是否有控制没有填写评论无法提交的配置项，我想在用户未填写评论内容时无法提交评论。

**Comment by lizheming** [comment]
1. 欢迎 PR
2. 现在评论内容为空，点提交是不会触发的

**Reply by Andy-njupt** [reply]
好的，感谢，但是目前未登录时是不能提交空内容，登陆后仍可提交空内容

**Reply by lizheming** [reply]
这个可以优化下

**Reply by Andy-njupt** [reply]
好的 感谢
